react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export default class ToggleSwitch extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {isClicked: false};
+        this.handleClick = this.handleClick.bind(this);
+
+    }
+
+    handleClick(){
+        this.setState({isClicked: true });
+    }
+
+    render(){
+        return(
+            <div>
+                <input type ={'checkbox'} className={'switchButton'} checked={this.state.handleClick} onChange={this.handleClick}>
+                </input>
```

Comment:
Remember `input` elements don't have closing tags. In react for any element that doesn't have a closing tag we self close, for example:

```JSX
<input className="my-input" />
```